packages/components-js/tests/e2e/puppeteer/specs/__snapshots__/default-dom.e2e.ts.snap

```diff
@@ -137,7 +137,7 @@ exports[`should have no basic DOM regression for p-flex-item 1`] = `
 `;
 
 exports[`should have no basic DOM regression for p-flyout 1`] = `
-"<div class="root" role="dialog" aria-modal="true" aria-hidden="true" tabindex="-1">
+"<div class="root" role="dialog" aria-modal="true" aria-hidden="true" tabindex="-1" inert="">
```

Comment:
Could you get rid of the empty value? `inert` has no value at all and it looks a bit weird with empty "".

Reply:
I don't think so. That's in the nature of jsx.